boards/arm/nrf9160_pca10090/doc/nrf9160_pca10090.rst

```diff
@@ -0,0 +1,207 @@
+
+Connections and IOs
+===================
+
+LED
+---
+
+* LED1 (green) = P0.2
+* LED2 (green) = P0.3
+* LED3 (green) = P0.4
+* LED4 (green) = P0.5
+
+Push buttons and Switches
+-------------------------
+
+* BUTTON1 = P0.6
+* BUTTON2 = P0.7
+* SWITCH1 = P0.8
+* SWITCH2 = P0.9
+* BOOT = SW5 = boot/reset
+
+Security components
+===================
+
+- Implementation Defined Attribution Unit (`IDAU`_).  The IDAU is implemented
+  with the System Protection Unit and is used to define secure and non-secure
+  memory maps.  By default, all of the memory space  (Flash, SRAM, and
+  peripheral address space) is defined to be secure accessible only.
+- Secure boot.
+
+
+Programming and Debugging
+*************************
+
+nrf9160_pca10090 supports the Armv8m Security Extension, and by default boots
+in the Secure state.
+
+Building Secure/Non-Secure Zephyr applications
+==============================================
+
+The process requires the following steps:
+
+1. Build the Secure Zephyr application using ``-DBOARD=nrf9160_pca10090`` and
+``CONFIG_TRUSTED_EXECUTION_SECURE=y`` in the the application project configuration file.
+2. Build the Non-Secure Zephyr application using ``-DBOARD=nrf9160_pca10090ns``.
+3. Merge the two binaries together.
+
+When building a Secure/Non-Secure application, the Secure application will
+have to set the IDAU (SPU) configuration to allow Non-Secure access to all
+CPU reources utilized by the Non-Secure application firmware. SPU
+configuration shall take place before jumping to the Non-Secure application.
+
+Building a Secure only application
+==================================
+
+Build the Zephyr app in the usual way, using ``-DBOARD=nrf9160_pca10090``.
+
+
+Flashing
+========
+
+Follow the instructions in the :ref:`nordic_segger` page to install
+and configure all the necessary software. Further information can be
+found in :ref:`nordic_segger_flashing`. Then build and flash
+applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Here is an example for the :ref:`hello_world` application.
+
+First, run your favorite terminal program to listen for output.
+
+.. code-block:: console
+
+   $ minicom -D <tty_device> -b 115200
+
+Replace :code:`<tty_device>` with the port where the board nRF9160 DK
+can be found. For example, under Linux, :code:`/dev/ttyACM0`.
+
+Then build and flash the application in the usual way.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: nrf9160_pca10040
+   :goals: build flash
+
+Debugging
+=========
+
+Refer to the :ref:`nordic_segger` page to learn about debugging Nordic boards with a
+Segger IC.
+
+
+Testing the LEDs and buttons in the nRF9160 DK
+**********************************************
+
+There are 2 samples that allow you to test that the buttons (switches) and LEDs on
+the board are working properly with Zephyr:
+
+.. code-block:: console
```

Comment:
Replace this code-block and body with a couple of bullet item with links to the sample docs:
```
* :ref:`blinky-samle`
* :ref:`button-sample`
```

Reply:
Yes, I am fixing this as well.

Reply:
I suppose "sample" instead of "samle" :)